### PR TITLE
Python Docker Image

### PR DESCRIPTION
**Pull request by Sushisource**

### CODE REVIEW

**Comment by Sushisource** [comment]
Go cloud is busted b/c schedules feat is not enabled for the NS. It passed Chad's PR b/c he was from a fork and we don't run Cloud on forks (we'll need to do PRs from not-forks).

I am working w/ saas to get the feature turned on.

File: harness/python/main.py [hunk 1]
```diff
@@ -45,11 +64,13 @@ async def run():
             raise ValueError(f"Cannot find registered feature for {rel_dir}")
         # Run
         try:
+            print("tls cfg", tls_config)
```
Comment (bergundy) [inline harness/python/main.py:67]: Yay, I have a comment on this PR

**Review by bergundy — APPROVED** [review]

**Comment by bergundy** [comment]
Might want to let Chad review but I didn't see anything that should block this PR

File: harness/python/main.py (hunk 1 above)
Comment (Sushisource) [reply harness/python/main.py:67]: Ah, thanks

**Comment by Sushisource** [comment]
Something is awry w/ Go still on Cloud even w/ schedules enabled. Bypassing it for now.

**Pull request merged**